ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form-tab-container.view.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Similar comment as before about logic abstraction for the almost fully duplicated code present in search-form, sharing, and result-form

Reply:
I recommend addressing this in a follow on ticket